src/components/ContactForm.js

```diff
@@ -10,18 +10,21 @@ const propTypes = {
     name: PropTypes.string,
     phoneNumber: PropTypes.string,
   })).isRequired,
-  onAddContact: PropTypes.func.isRequired,
+  onUpdateContact: PropTypes.func.isRequired,
 };
 
 class ContactForm extends Component {
   constructor(props) {
     super(props);
 
+    this.isEditMode = this.props.match.path.includes('/edit-contact/');
```

Comment:
It would be better to write a function to check than set it to component instance

Reply:
Replaced isEditMode property with isEditMode method